integration-cli/docker_cli_build_test.go

```diff
@@ -5608,30 +5608,6 @@ func (s *DockerSuite) TestBuildWithExtraHostInvalidFormat(c *testing.T) {
 
 }
 
-func (s *DockerSuite) TestBuildContChar(c *testing.T) {
```

Comment:
I don't like that we are removing the test.
Even if it is tested in buildkit, this is more of an end-to-end test, and we make sure it doesn't break regardless of builder.

Reply:
@cpuguy83 who writes a `\` as the last character of a Dockerfile? I am of the opinion that it's such an edge case that I'm prioritizing removing legacy tests that are useless. But if you feel strongly about this I can fix the test.